rtlib/src/LandmarkIO.cpp

```diff
@@ -31,5 +32,7 @@ void LandmarkIO::read_landmarks_file_() {
 
 void LandmarkIO::clear_() {
     /** TODO: Figure out clearing mechanism for ITK::LandmarkPointContainer */
+    fixedLandmarks_ = LandmarkContainer::New();
+    movingLandmarks_ = LandmarkContainer::New();
```

Comment:
This can happen in `generate_affine_transform_` since you're not really clearing. You're just prepping a new object for output. Also, since these will be smart pointers, it's best to continue making new objects (as opposed to clearing the containers) so that any previously created pointers aren't suddenly pointing to incorrect data.